tracer.go

```diff
@@ -347,6 +356,27 @@ func (t *Tracer) Tags() []opentracing.Tag {
 	return tags
 }
 
+// getTag returns the value of specific tag, if not exists, return nil.
+func (t *Tracer) getTag(key string) interface{} {
```

Comment:
it should return `(interface{}, bool)` to indicate found / not-found, like map does.